src/index.html

```diff
@@ -6,7 +6,49 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_30 calendar--start-day_tue">
```

Comment:
this is because the class name here and the one in the loop are different
here you wrote `calendar--month-length_#`, but in the loop in calendar.scss file your class name is `calendar--month-length--30`. change it either here or there